libvast/fbs/value_index.fbs

```diff
@@ -17,7 +17,11 @@ table LegacyValueIndex {
   /// because we can't distinguish `0` from "Field doesn't exist", but that's fine
   /// here because the partition itself will always occupy index 0 of the external
   /// container.
-  external_container_idx: ulong;
+  external_container_idx_caf_0_17: ulong;
+
+  caf_0_18_data: [ubyte];
+
+  external_container_idx_caf_0_18: ulong;
```

Comment:
Please also add comments along the lines of `/// NOTE: This can be marked (deprecated) once the minimum partition version is 3.` to the two older fields, and please copy over the comments from the two old fields to the new ones. They get copied over by the FlatBuffers compiler into the generated code, so they show up in auto-completion and can be really helpful.